doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,90 @@
+================
+Cross Validation
+================
+
+Cross validation allows to estimate an algorithm's performance on unseen data.
+
+It is based on training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and and all validation
+results are combined, e.g. by averaging.
+
+One type of cross validation is :math:`k-`fold cross validation.
+
+Suppose we have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{N}`
+for a positive integer :math:`N`, then :math:`k-`fold cross validation partitions
+the data into :math:`P_1, P_2, ..., P_n` each with approximately equal size. Next, we train the model on
```

Comment:
pls start new sentences in a new line, makes later editing much easier

Reply:
Sure, will be fixed in the next commit. 

So by this, do you mean:

Sentence 1 .....
Sentence 2 .....

or 

Sentence 1....

Sentence 2....

?